e2etest/rewarding_test.go

```diff
@@ -125,7 +125,7 @@ func TestBlockReward(t *testing.T) {
 
 func TestBlockEpochReward(t *testing.T) {
 	// TODO: fix the test
-	t.Skip()
+	//t.Skip()
```

Comment:
why?

Reply:
incorrect operation

Reply:
https://golang.org/pkg/testing/#hdr-Skipping
test skipping is supported in golang std. Please check you go environment. This should not be an incorrect operation.

Reply:
It was opened when I tested the project. It was my personal misoperation

Reply:
sorry

Reply:
Recovered